pkg/operator/observe_config.go

```diff
@@ -24,11 +24,13 @@ import (
 	"k8s.io/client-go/util/flowcontrol"
 	"k8s.io/client-go/util/workqueue"
 
+	"github.com/openshift/api/operator/v1alpha1"
 	operatorconfigclientv1alpha1 "github.com/openshift/cluster-kube-apiserver-operator/pkg/generated/clientset/versioned/typed/kubeapiserver/v1alpha1"
 	kubeapiserveroperatorinformers "github.com/openshift/cluster-kube-apiserver-operator/pkg/generated/informers/externalversions"
+	"github.com/openshift/library-go/pkg/operator/v1alpha1helpers"
 )
 
-type observeConfigFunc func(kubernetes.Interface, *rest.Config, map[string]interface{}) (map[string]interface{}, error)
+type observeConfigFunc func(kubernetes.Interface, *rest.Config, map[string]interface{}) (map[string]interface{}, []error)
```

Comment:
name the args.  I think that map is `previousObservedConfig`

Reply:
> name the args. I think that map is `previousObservedConfig`

Oh, it isn't.  You need to add that one to build this method correctly.

Reply:
The observed config being composed was initialized with the previously observed config, so no need to pass the previously observed config separately.